src/sql/workbench/parts/notebook/browser/notebook.contribution.ts

```diff
@@ -160,6 +160,19 @@ configurationRegistry.registerConfiguration({
 	}
 });
 
+configurationRegistry.registerConfiguration({
+	'id': 'notebook',
+	'title': 'Notebook',
+	'type': 'object',
+	'properties': {
+		'notebook.sqlKernelStopOnError': {
+			'type': 'boolean',
+			'default': true,
+			'description': localize('notebook.sqlKernelStopOnError', "SQL kernel: stop notebook execution when error occurs in a cell.")
```

Comment:
>SQL kernel: stop notebook execution when error occurs in a cell. [](start = 61, length = 64)

"SQL kernel: Stop notebook execution when an error occurs in a cell."

We also capitalize Notebook in other places, should you do that here too?

Reply:
thanks